MessengerOnFileForLinux/Common/test/TimeManagmentTest.cpp

```diff
@@ -22,7 +22,21 @@ TEST(ConvertTimeToString, TimeFormatIsCorrect)
 
 TEST(ConvertTimeToString, TimeValueIsCorrect)
 {
-    const auto unixEpochTimeAsString = TimeManagment::convertTimeToString(std::time_t{});
-    std::string expectedTimeValue = "1970-01-01 01:00:00";
-    EXPECT_EQ(unixEpochTimeAsString, expectedTimeValue);
+    constexpr auto day = 9;
+    constexpr auto month = 2;
+    constexpr auto year = 2019;
+    std::tm timePoint =
+    {
+        .tm_sec  = 0,
+        .tm_min  = 0,
+        .tm_hour = 0,
+        .tm_mday = (day),
+        .tm_mon  = (month) - 1,
+        .tm_year = (year) - 1900,
+    };
```

Comment:
unclear way of timePoint initialization. Also imho it should be in .hpp not in TEST

Reply:
still I think that it's unclear and should not be here

Reply:
May will i create function in TimeManagment module createTimePoint()?

Reply:
You could write what exactly you mean?

Reply:
Use other way of initialization.